src/ferui-components/tree-view/tree-view-component.ts

```diff
@@ -146,10 +149,16 @@ export class FuiTreeViewComponent<T> implements OnInit, OnDestroy {
           return this.createTreeNode(child, null);
         });
         this.scrollViewArray = this.nonRootArray;
+        if (this.autoSelector) {
+          this.callAutoSelector(this.scrollViewArray.map(it => it.data));
+        }
       });
     } else {
       this.rootNode = this.createTreeNode(this.treeNodeData, null);
       this.scrollViewArray = [this.rootNode];
+      if (this.autoSelector) {
```

Comment:
you also need to check that `this.scrollViewArray` is not empty too.

Reply:
Done